src/main/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceService.scala

```diff
@@ -507,11 +507,9 @@ class ResourceService(
           // resources with auth domains logically can't have public policies but also technically allowing them poses a problem
           // because the logic for public resources is different. However, sharing with the auth domain should have the
           // exact same effect as making a policy public: anyone in the auth domain can access.
-          if (public)
```

Comment:
why was this removed? based on the function description seems useful to have

Reply:
good catch! this is probably from bad merge